gateway-server/server.py

```diff
@@ -0,0 +1,91 @@
+from flask import Flask, request, jsonify, render_template
+import requests
+# TODO: from kazoo.client import KazooClient
+from pymongo import MongoClient
+import random
+
+app = Flask(__name__)
+
+# TODO: Add Zookeeper functionality
+
+mongo_uri = '<hidden-for-pull-request>'
+client = MongoClient(mongo_uri)
+db = client['nabu']
+
+# List of IPFS node addresses
+IPFS_URL = [
+    'http://localhost:5001',
+    # TODO: Add more IPFS node addresses
+]
```

Comment:
This could be a good candidate for Kazoo - each IPFS node could register w/ ZooKeeper discovery and this service could query that to get the list dynamically. Maybe that was the intent.

Reply:
That's a good point! I think we need the Nabu peer to create the znode? That way the znode gets deleted when the Nabu peer is down.

Reply:
I don't think nabu peers should be discovered using ZooKeeper. That may beat the purpose of decentralization. The ZooKeeper instance may be centrally owned and may block new peers. Also, there can be millions of peers which would be discovered by millions of clients. Not sure if a centralized coordinator would scale well.

Having bootstrap nodes is the only way to discover all other Nabu peers. Unfortunately, that is not possible with Nabu. So, we can hardcode for now and I can expose a health check API that the gateway can use to ping and make sure the service is available.